src/components/SchemaView/SchemaViewMain.vue

```diff
@@ -673,6 +672,16 @@ export default {
       this.clickedLabel = newLabel;
     },
 
+    renameTable(oldLabel, newLabel) {
+      this.$emit("renameTable", oldLabel, newLabel);
+      this.clickedLabel = newLabel;
+      this.settingsStore.renameNodeTable(oldLabel, newLabel);
```

Comment:
This should not be done. Instead, you should follow the approach of `updatePlaceholderNodeTableLabel`.